app/pages/preview/template.test.js

```diff
@@ -7,7 +7,7 @@ const setup = {
 };
 
 describe('preview page', () => {
-  it('should render the title of the preview page', createTestHarness(setup, (harness) => {
+  it('should render the viwSolution component', createTestHarness(setup, (harness) => {
```

Comment:
do we need a test to check a section is rendered?

Reply:
I just checked something would be rendered from the coponent